lib/elevio/index.js

```diff
@@ -0,0 +1,71 @@
+
+var integration = require('analytics.js-integration');
+var tick = require('next-tick');
+
+/**
+ * Expose `Elevio` integration.
+ */
+
+var Elevio = module.exports = integration('Elevio')
+  .assumesPageview()
+  .option('accountId', '')
+  .global('_elev')
+
+/**
+ * Initialize elevio.
+ *
+ * @param {Facade} page
+ */
+
+Elevio.prototype.initialize = function(page){
+  var self = this;
+  this.load(function(){
+    tick(self.ready);
+  });
+}
+
+Elevio.prototype.load = function(callback){
+    window._elev = window._elev || {};(function(){var i,e;i=document.createElement("script"),i.type='text/javascript';i.async=1,i.src="https://static.elev.io/js/v3.js",e=document.getElementsByTagName("script")[0],e.parentNode.insertBefore(i,e);})();
+    // window._elev = window._elev || {};(function(){var i,e;i=document.createElement("script"),i.type='text/javascript';i.async=1,i.src="http://widget.elev.dev:8000/scripts/v3.js",e=document.getElementsByTagName("script")[0],e.parentNode.insertBefore(i,e);})();
+    _elev.account_id = this.options.accountId;
+    _elev.segment = true;
+    callback();
+};
```

Comment:
looks like we can we move this into the initialize method.

you can declare your script with the `.tag()` static method on the integration constructor, and then `this.load()` will download the tag accordingly.

With that in mind, can we give this a try in lieu of overriding the load method? (I haven't tested but should work :)

```
/**
 * Expose `Elevio` integration.
 */

var Elevio = module.exports = integration('Elevio')
  .assumesPageview()
  .option('accountId', '')
  .global('_elev')
  .tag('<script src="https://static.elev.io/js/v3.js">');

/**
 * Initialize elevio.
 *
 * @param {Facade} page
 */

Elevio.prototype.initialize = function(page){
  window._elev = window._elev || {};
  _elev.account_id = this.options.accountId;
  _elev.segment = true;
  this.load(self.ready);
}
```